README.md

```diff
@@ -649,6 +652,62 @@ And then configure the library to use it:
+
+In these cases, it's better to directly control how FunWithFlags starts its processes.
+
+The first step is to add the `FunWithFlags.Supervisor` module directly to the supervision tree of the host application. For example, in a Phoenix app it would look like this:
+
+```diff
+defmodule MyPhoenixApp.Application do
+  @moduledoc false
+  use Application
+
+  def start(_type, _args) do
+    children = [
+      MyPhoenixApp.Repo,
+      MyPhoenixAppWeb.Telemetry,
+      {Phoenix.PubSub, name: MyPhoenixApp.PubSub},
+      MyPhoenixAppWeb.Endpoint,
++     FunWithFlags.Supervisor,
+    ]
+
+    opts = [strategy: :one_for_one, name: MyPhoenixApp.Supervisor]
+    Supervisor.start_link(children, opts)
+  end
+
+  # ...
+```
+
+Then it's necessary to configure the Mix project to not start the `:fun_with_flags` application automatically. This can be accomplished in the Mixfile in a number of ways, for example: (**Note**: These are alternative solutions, you don't need to do both. You must decide which is more appropriate for your setup.)
+
+* Declare the `:fun_with_flags` dependency with either the `runtime: false` or `app: false` options. ([docs](https://hexdocs.pm/mix/1.11.3/Mix.Tasks.Deps.html#module-dependency-definition-options))
+
+```diff
+- {:fun_with_flags, "~> 1.6"},
++ {:fun_with_flags, "~> 1.6", runtime: false},
+```
+
+* Declare that the `:fun_with_flags` application is managed directly by your host application ([docs](https://hexdocs.pm/mix/1.11.3/Mix.Tasks.Compile.App.html)).
+
+```diff
+  def application do
+    [
+      mod: {MyPhoenixApp.Application, []},
++     included_applications: [:fun_with_flags],
+      extra_applications: [:logger, :runtime_tools]
+    ]
+  end
+```
+
+The result of those changes is that the `:fun_with_flags` application won't be loaded and started automatically, and therefore the FunWithFlags supervision tree won't risk to be started before the other processes in the host Phoenix application. Rather, the supervision tree will start alongside the other core Phoenix processes.
+
+One final note on this topic is that if you're also using [`FunWithFlags.UI`](https://github.com/tompave/fun_with_flags_ui) (refer to the [Web Dashboard](#web-dashboard) section, above in this document), then that will need to be configured as well. The reason is that `:fun_with_flags` is a dependency of `:fun_with_flags_ui`, so including the latter as a dependency will cause the former to be auto-started despite the configuration described above. To avoid this, the same configuration should be used for the `:fun_with_flags_ui` dependency, regardless of the approach used (`runtime: false`, `app: false`, or `included_applications`).
```

Comment:
```suggestion
One final note on this topic is that if you're also using [`FunWithFlags.UI`](https://github.com/tompave/fun_with_flags_ui) (refer to the [Web Dashboard](#web-dashboard) section, above in this document), then that will need to be configured as well. The reason is that `:fun_with_flags` is a dependency of `:fun_with_flags_ui`, so including the latter as a dependency will cause the former to be auto-started despite the configuration described above. To avoid this, the same configuration should be used for the `:fun_with_flags_ui` dependency, regardless of the approach used (Option A (`runtime: false`, `app: false`), or Option B (`included_applications`)).
```

Reply:
Makes sense, but I'd rather avoid the nested parentheses.
```suggestion
One final note on this topic is that if you're also using [`FunWithFlags.UI`](https://github.com/tompave/fun_with_flags_ui) (refer to the [Web Dashboard](#web-dashboard) section, above in this document), then that will need to be configured as well. The reason is that `:fun_with_flags` is a dependency of `:fun_with_flags_ui`, so including the latter as a dependency will cause the former to be auto-started despite the configuration described above. To avoid this, the same configuration should be used for the `:fun_with_flags_ui` dependency, regardless of the approach used (Option A: `runtime: false`, `app: false`; or Option B: `included_applications`).
```

Reply:
Agreed! I didn't like the double parens either. Nice change!